config/initializers/constants.rb

```diff
@@ -28,3 +28,18 @@
 
 VOLUNTEERS_REQUIRED_FOR_FUNDING = 50
 MAX_VOLUNTEERS_FOR_HIGHLIGHT_OFFER = 30
+
+RESOURCE_ANNOUNCEMENTS = [
```

Comment:
Why not have these too inside the controller. While we moved the global announcements in the controller, it makes sense to have these too in the `OffersController`.